lib/commands/recordscreen.js

```diff
@@ -210,6 +215,37 @@ commands.startRecordingScreen = async function (options = {}) {
       `The value of '${timeLimit}' has been passed instead.`);
   }
 
+  let {
+    mjpegServerScreenshotQuality,
+    mjpegServerFramerate,
+  } = await this.proxyCommand('/appium/settings', 'GET');
+  if (videoQuality) {
+    const quality = _.isInteger(quality) ? quality : QUALITY_MAPPING[_.toLower(videoQuality)];
+    if (!quality) {
+      throw new Error(`videoQuality value should be one of ${JSON.stringify(_.keys(QUALITY_MAPPING))} or a number in range 1..100. ` +
+        `'${videoQuality}' is given instead`);
+    }
+    mjpegServerScreenshotQuality = mjpegServerScreenshotQuality !== quality ? quality : undefined;
+  } else {
+    mjpegServerScreenshotQuality = undefined;
```

Comment:
I think this is redundant.

Reply:
not really. This defines if settings update should be called. `undefined` value means no change is needed to the current value

Reply:
I could set it to null, but then it will be necessary to make the `setSettings` call more complicated, since we'll have to check which value should be excluded from the arguments dictionary. Passing `undefined` does the job automatically.

Reply:
ok, makes sense